styles.css

```diff
@@ -18,6 +18,19 @@ p {
     background-color: #1BAB49;
 }
 
+.ingredient {
+    border-radius: 25%;
```

Comment:
The styles here need a little tweaking. Change the border-radius to a concrete pixel value, and check the padding on text and other elements.
Also, it might be nice to have the boxes change colour when they're clicked? You can do it with the :checked css selector on the label class. (maybe all the styling can go on the label if you give it the ingredient class, removing the need for the outer div!)